server/src/resolvers/tutorium/create.ts

```diff
@@ -58,8 +63,24 @@ export async function createTutorium (args: TutoriumCreateInput, context: Contex
       }
     }
 
+    //self added:
+    if (args.tutorId.length < 1) {
+      return {
+        errors: [{
+          code: TutoriumCreateErrorCode.TUTOR_NOT_VALID,
+          message: 'The provided Tutor is not valid'
+        }]
+      }
+    }
+
     const tutorium = new Tutorium()
     tutorium.name = args.name
+    const tutor = await User.findOne(args.tutorId)//set Tutor via ID
```

Comment:
Der Kommentar kann bleiben, aber bitte in die Zeile darüber und mit Leerzeichen nach dem //